recipes/pep257/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.7.0" %}
+{% set hash_type = "md5" %}
+{% set hash = "ff3dc9622c2406b7b583a283fbea651e" %}
+
+package:
+  name: pep257
+  version: {{ version }}
+
+source:
+  fn: pep257-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/p/pep257/pep257-{{ version }}.tar.gz 
+  {{ hash_type }}: {{ hash }}
+
+build:
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - pep257
+
+about:
+  home: https://github.com/GreenSteam/pep257
+  license: MIT License
+  summary: 'Python docstring style checker (replaced by pydocstyle)'
+  description: |
+    pep257 is a static analysis tool for checking compliance with Python
+    PEP 257.
+    The framework for checking docstring style is flexible, and custom checks
+    can be easily added, for example to cover NumPy docstring conventions.
+    pep257 supports Python 2.6, 2.7, 3.2, 3.3, 3.4, pypy and pypy3.
+  doc_url: http://pep257.readthedocs.io/en/latest
+  dev_url: http://github.com/GreenSteam/pep257
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - Nurdok
```

Comment:
@Nurdok, is it alright if include you on `pep257`?

We are aware it is being phased out and replaced by `pydocstyle`, but some packages have yet to update the dependency. So, we would like to keep it around ATM for that reason. As a result, we expect the effort spent here to be negligible.